pkg/kube/kclient/client_test.go

```diff
@@ -280,6 +280,7 @@ func TestErrorHandler(t *testing.T) {
 	c.Kube().(*fake.Clientset).Fake.PrependReactor("*", "*", func(action k8stesting.Action) (bool, runtime.Object, error) {
 		return true, nil, fmt.Errorf("nope, out of luck")
 	})
+	c.RunAndWait(test.NewStop(t))
```

Comment:
why is this needed? seems like it should be valid to make a client without running?

Reply:
After the queue used internally by `NewFakeClient` is initialized, a new goroutine will be used to automatically start an infinite for loop, and a signal is required to stop it. The purpose here is to pass a stop signal so that it can exit, otherwise the checker says the goroutine is leaking.

https://github.com/kubernetes/client-go/blob/master/util/workqueue/delaying_queue.go#L141

L111 and L121
https://github.com/istio/istio/blob/master/pkg/kube/controllers/queue.go#L111

Reply:
> seems like it should be valid to make a client without running?

It looks like L280's ReactionFunc is working, rather than `NewFakeClient` not running.